### ISSUE TITLE
texture issue (1.16.4)

### ISSUE THREAD

**Issue by ZoruaBit** [HellFirePvP/AstralSorcery#1617]
a simple texture issue when pushing starmetal ore with a vinilla piston
![image](https://user-images.githubusercontent.com/39221658/103455818-f8afe280-4cb5-11eb-9879-755f62499541.png)

**Comment by HellFirePvP** [comment]
This is a forge bug introduced with the fix of https://github.com/MinecraftForge/MinecraftForge/issues/6915 for multi-layer block models.
I'll open an issue on their end with the steps to reproduce on a more generic level.

Sadly, this is nothing i specifically can fix, though this will occur with any mod's blocks using forge's multi-layer models.

**Comment by HellFirePvP** [comment]
This is waiting on being fixed/merged into forge: https://github.com/MinecraftForge/MinecraftForge/pull/7579

**Comment by ZoruaBit** [comment]
okey dokey. thank you. it isnt really a big issue in all honisty.

**Comment by HellFirePvP** [comment]
Am gonna close this here for now as it's not my issue specifically. For the fix, have an eye on the Forge PR.
